wdn/templates_5.3/js-src/banner.babel.js

```diff
@@ -0,0 +1,45 @@
+// WDN Notice Banner Display
+function displayWDNNoticeBanner() {
+    var bannerEnabled = true;
+    if (bannerEnabled) {
+        var messageContent = false;
+        var messageSessionKey = 'wndNoticeMessage';
```

Comment:
we have a typo here: 'wnd' that's never gotten fixed. let's change this.

Reply:
fixed